assets/tsconfig.json

```diff
@@ -9,7 +9,7 @@
   ],
   "compilerOptions": {
     /* Basic Options */
-    "target": "es5",                       /* Specify ECMAScript target version: 'ES3' (default), 'ES5', 'ES2015', 'ES2016', 'ES2017','ES2018' or 'ESNEXT'. */
+    "target": "es6",                       /* Specify ECMAScript target version: 'ES3' (default), 'ES5', 'ES2015', 'ES2016', 'ES2017','ES2018' or 'ESNEXT'. */
```

Comment:
Question: what change makes this update necessary? It looks like es6 has [enough browser support](https://caniuse.com/?search=es6) that we can safely use it, but I'm curious about the exact reason.

Reply:
At one stage of this PR I was running into an error instantiating `RecenterControl`, which this update seemed to resolve. May have been a confounding change that actually did the trick though since resetting to "es5" did not resurface the old error.